### DISCUSSION TITLE
Simple AI for recommendations

### DISCUSSION THREAD

**Discussion by ghost** [Hanziness/AnotherPomodoro#194] [Ideas & feature requests]
He could analyze [patterns ](https://github.com/Hanziness/AnotherPomodoro/discussions/190)and [history ](https://github.com/Hanziness/AnotherPomodoro/discussions/184)to provide recommendations for adjusting the user's rhythm. And potentially create new cycles regarding the days and hours he discovered regarding history.

**Comment by Hanziness** [comment]
Thank you for the idea! I was thinking on doing something like this, but only as long as it can be executed in the browser on the user's side (this is a statically served webpage, I would not like to put an extra back-end behind it). It's quite behind on the backlog as simple usage statistics should be implemented first.
